docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,254 @@ grand_parent: "General"
 ---
 
 # Getting Started With bpmn-js
+
+bpmn-js is a [BPMN 2.0](https://www.omg.org/spec/BPMN/2.0/About-BPMN/) rendering toolkit and web modeler. It is written in JavaScript, embeds BPMN 2.0 diagrams into modern browsers and requires no server backend. That's why it is easy to embed it into any web application.
+
+Feel free to play with it [on our demo playground](https://demo.bpmn.io/s/start).
+
+![image](/assets/images/bpmn-start.gif)
+
+bpmn-js can be included in any application with:
+
+* an [all-in-one pre-packaged version of the library](https://unpkg.com/browse/bpmn-js/dist/)
+* the [npm](http://npmjs.com/package/bpmn-js) version, which gives you access to individual library components and allows for easier extensibility
+
+## Including the Root Element
+
+bpmn-js requires a root DOM element (for instance, a simple DIV element) on which the rendering is performed. We should simply create the root DOM element in the HTML file:
+
+```html
+<!-- BPMN diagram container -->
+<div id="canvas"></div>
+```
+
+Here, we've given an ID to the root DOM element since we'll be using this ID to select this component and pass to bpmn-js later on.
+
+## Including the Stylesheet
+
+We should include [diagram-js](/general/getting-started/diagram-js) CSS and [BPMN Icon Font](https://github.com/bpmn-io/bpmn-font) in order to ensure that everything looks good. There're several ways we can achieve that:
+
+We can include the CSS via `link` tag in our HTML file. Note that these CSS files are shipped with with the bpmn-js distribution under [dist/assets folder](https://unpkg.com/browse/bpmn-js/dist/assets/).
+
+```html
+<link rel="stylesheet" href="bpmn-js/dist/assets/diagram-js.css" />
+<link rel="stylesheet" href="bpmn-js/dist/assets/bpmn-font/css/bpmn.css" />
+```
+
+If we've install the bpmn-js via [NPM](#including-via-npm), we can additionally include these CSS files using `import` statement in our Javascript file:
+
+```javascript
+import 'bpmn-js/dist/assets/diagram-js.css';
+import 'bpmn-js/dist/assets/bpmn-font/css/bpmn-embedded.css';
+```
+
+## Alternatives to Use bpmn-js
```

Comment:
Weird title. It sounds like you're going to suggest alternative libraries. We should explain that there are 3 pre-made packages of bpmn-js that include different features and that the users can create their own.

Reply:
Good point. I changed the title to: `bpmn-js Use Cases`